package.json

```diff
@@ -12,12 +12,20 @@
     "build:client": "cross-env NODE_ENV=production webpack --config webpack/config.prod.js",
     "build:server": "cross-env NODE_ENV=production webpack --config webpack/config.server.js",
     "build:examples": "cross-env NODE_ENV=production webpack --config webpack/config.examples.js",
-    "test": "npm run lint",
+    "test": "jest",
```

Comment:
Yes yes I totally missed the `.travis.yml` file.

I am more leaning toward adding a new item there, lets say `test:ci`, to be called from the travis file [here](https://github.com/processing/p5.js-web-editor/blob/fc8e6144a112cffbf6b0326a950f2834f0f93ba4/.travis.yml#L20) and leave `test` to call `jest` as it is right now, just to have the names to be more descriptive.

For the `test:ci` I think it is fine to call both lint and jest now (`npm run lint && jest`) since the only unit test is passing. Also tests are going to be *in flux* for a while and the best way to make sure it gets better is to be committed to it as soon as possible. 

Anyways it is a suggestion. Your call.

Reply:
cool, i like that! just want to make sure that this doesn't accidentally break the travis build.

Reply:
Yeah it worked on Travis! See https://travis-ci.org/processing/p5.js-web-editor/builds/448056217, or if you scroll to the bottom of this PR to the section that says "All checks have passed", you can check the latest Travis build on this PR there.